frontend/components/ProfilePublication/Publication.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+
+class Publication extends Component {
+  render() {
+    return (
+      <>
+        {/* {style} */}
+        <a
+          href="{this.props.data.link}"
```

Comment:
I think something is wrong here. It should be `href={this.props.data.link}` . without the quotes because that will signify as a string

Reply:
yes definitely the link won't work

Reply:
Okay removed. That was written quite a long time ago. Didn't update it and now removed. Thanks for telling me